### DISCUSSION TITLE
I always fail to deploy my projects on vercel it gives me error 404 every time

### DISCUSSION THREAD

**Discussion by Heroshi1947** [vercel/community#7301] [Help]
### Summary

 i am sick can you guys tell what i am doing wrong. 
I create a project it runs smooth on my system then i upload it on github and connect to vercel but it doesnt work 
Am i creating folder structure wrong ? 
I generally create a Main app folder having backend files but i keep frontend code in client folder within main folder 
is vercel not recognising it ? 
or should i change some settings during deployment 
I am using MERN stack for project 

### Example

https://you-blog-lake.vercel.app/

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/123864184/fde5abd7-faab-4c48-9488-af84b36b774b)
this is my file structure the client folder is having React files of frontend
all others folders and files are concerned with backend

**Comment by pawlean** [comment]
Hi @Heroshi1947!

Maybe **[this discussion](https://github.com/orgs/vercel/discussions/7285)** from yesterday will help? Specifically [my comment](https://github.com/orgs/vercel/discussions/7285#discussioncomment-9883133):

> Can you try setting the Root Directory to web as this is where the application code lives?

Except put yours as client as where that's where all your application code is?

**Reply by Heroshi1947** [reply]
thank you for your suggestion mam! 
in the discussion the person was overriding the deafult commands 
i dont override those in my project i just go by what vercel defaults 
is that fine ?

**Reply by pawlean** [reply]
@Heroshi1947 Yes, that's fine. It depends on your project 😃

[a comment by an automated account is not shown]
